osc/git_scm/store.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, path, check=True):
 
         # TODO: how to determine if the current git repo contains a project or a package?
         self.is_project = False
-        self.is_package = os.path.exists(os.path.join(self.abspath, ".git"))
+        self.is_package = os.path.exists(os.path.join(self.abspath, ".git")) or os.path.exists(os.path.join(self.abspath, "../.git"))
```

Comment:
Are you really suggesting that package is every directory that has .git in the parent dir?
We should rather find a better way of identifying project and package git checkouts.